scripts/setup_k8s_env.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+sudo pip3 install tensorflow
```

Comment:
Why is this needed for status validation?

Reply:
Because   the module https://github.com/sql-machine-learning/elasticdl/blob/492fe94a377ff15cb67468fefbc99f89a16147a2/elasticdl/python/common/k8s_client.py#L13 in `elasticdl.python.common.k8s_client.Client`  imports Tensorflow and I import `elasticdl.python.common.k8s_client.Client` in `validate_jo_status.py`.

Reply:
I find dependency packages for `elasticdl.python.common.k8s_client.Client` is too many. Tensorflow, odps and others. So, it is more simple to use an indenpent `Client` for `validate_job_status.py`. I will close the PR.